docs/docs/admin/installation.mdx

```diff
@@ -186,3 +186,92 @@ Then point your Ingress to the Anubis port:
               # diff-add
               name: anubis
 ```
+
+## Nginx
+
+The below is a very minimal Nginx config example to demonstrate how to
+insert Anubis in the middle of a standard nginx configuration.
+Do not forget to add `access_log`, `server_tokens`, etc. as usual.
+Also one does have to correct the paths, eg for SSL keys/certs.
+
+First we define `conf-anubis.inc` where we include standard Anubis configuration:
+```
+# Serve Anubis images directly instead of having it pass through to the service
```

Comment:
That is an absolute micro optimisation which is not worth it, since right now anibus proxies all actual content anyway and the assets are tiny.